desktop-widgets/filterwidget2.cpp

```diff
@@ -1,8 +1,25 @@
-#include "filterwidget2.h"
+#include "desktop-widgets/filterwidget2.h"
+#include "desktop-widgets/simplewidgets.h"
+
+#include <QDoubleSpinBox>
 
 FilterWidget2::FilterWidget2(QWidget* parent)
 : QWidget(parent)
 , ui(new Ui::FilterWidget2())
 {
 	ui->setupUi(this);
+
+	connect(ui->maxRating, &StarWidget::valueChanged, [this]{ updateFilter();});
```

Comment:
yuck that's ugly. Come on, @tcanabrava - you know I hate this stuff. Just use a function pointer.